src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -248,29 +250,44 @@ def add_summaries(
         self,
         summaries: list[SingleBeadSummary | PorositySummary | MicrostructureSummary],
         iteration: int = DEFAULT_ITERATION,
-    ):
+    ) -> int:
         """Add summaries of executed simulations to the parametric study.
```

Comment:
```suggestion
        """Add summaries of completed simulations to the parametric study.
```

Reply:
Safee, does executed mean completed? If so, use completed instead. If executed means started or initiated, but they may or may not be completed yet, then use initiated. I'll try to think of a better word but executed is not great.

Reply:
We should change it to completed since it represents the status of the simulation. An executed simulation can either be completed or errored.